docs/guides/api.en.md

```diff
@@ -345,7 +471,344 @@
+    Msl.AddReferenceTable(
+        nameObject:"o_bandit_goon_club", 
+        table:"barrelsSpecial", 
+        tiers: new tiers[] { {tier: 4, table:"goon"}, {tier: 7, table:"goon2"} }
+    );
+
+
+    // Example 4: Same as above, except ids 2334 and 2335 ignore the tier rules and are assigned to `goon3` and `goon4` respectively.
+    Msl.AddReferenceTable(
+        nameObject:"o_bandit_goon_club", 
+        table:"barrelsSpecial", 
+        tiers: new tiers[] { {tier: 4, table:"goon3"}, {id: 7, table:"goon4"} }, 
+        ids: new ids[] { {id: 2334, table:"goon"}, {id: 2335, table:"goon2"} }
+    );
+    ```
+    <h3>Arguments</h3>
+
+    | **Type** | **Name** | **Description**|
+    | :---: | :---: | :--- |
+    | `String` | `nameObject` | The name of the gameobject to assign to a loot table. |
+    | `String` | `table` | The name of the loot table to assign the gameobject to. |
+    | `Dict<int, string>` | `ids` | Overrides for specific ids. |
+    | `Dict<int, string>` | `tiers` | Overrides for specific tiers. |
+
+    <h3>Returns</h3>
+
+    | **Type** | **Description** |
+    | :--- | :--- |
+    | `N/A` | N/A |
+
+??? example "AddReferenceTableForMultipleObjects `method`"
+    <h3>Summary</h3>
+    Assigns multiple gameobjects to a loot table.</br>
+    This means that any and all instances of these gameobjects will drop items from the specified loot table.
+    <h3>Example</h3>
+    ```c#
+    // Assigns all instances of `o_bandit_goon_club` and `o_bandit_goon_cleaver` to the `barrelsSpecial` loot table.
+    Msl.AddReferenceTableForMultipleObjects(table: "barrelsSpecial", "o_bandit_goon_club", "o_bandit_goon_cleaver");
+    ```
+    <h3>Arguments</h3>
+
+    | **Type** | **Name** | **Description**|
+    | :---: | :---: | :--- |
+    | `String` | `table` | The name of the loot table to assign the gameobjects to. |
+    | `String[]` | `nameObjects` | The names of the gameobjects to assign to the loot table. |
+
+    <h3>Returns</h3>
+
+    | **Type** | **Description** |
+    | :--- | :--- |
+    | `N/A` | N/A |
+
+---
+
+## Rooms
+
+??? example "GetRoom `method`"
+    <h3>Summary</h3>
+    Gets a room from the game files by name.
+    <h3>Example</h3>
+    ```c#
+    Msl.GetRoom("r_myRoom")
+    ```
+    <h3>Arguments</h3>
+
+    | **Type** | **Name** | **Description**|
+    | :---: | :---: | :--- |
+    | `String` | `name` | The name of the room to get. |
+
+    <h3>Returns</h3>
+
+    | **Type** | **Description** |
+    | :--- | :--- |
+    | `UndertaleRoom` | The room if found. |
+
+??? example "GetRooms `method`"
+    <h3>Summary</h3>
+    Gets a list of all the rooms in the game.
+    <h3>Example</h3>
+    ```c#
+    Msl.GetRooms()
+    ```
+    <h3>Returns</h3>
+
+    | **Type** | **Description** |
+    | :--- | :--- |
+    | `IEnumerable<UndertaleRoom>` | An iterable of all the rooms in the game. |
+
+??? example "AddRoom `method`"
+    <h3>Summary</h3>
+    Adds a room to the game.
```

Comment:
maybe we should explain what happens if already existing

Reply:
Definitely.